aws/ecs_cluster.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// tagging an ECS cluster with a given tag key-value pair and a cluster ARN
+func tagEcsCluster(awsSession *session.Session, clusterArn *string, tagKey string, tagValue string) (*ecs.TagResourceOutput, error) {
```

Comment:
NIT: Unless you needed the returned data from creating tags (rare), you can probably return _just_ the `error`.

Reply:
good point, thank you :)